UnityProject/Assets/Scripts/Systems/DynamicAmbience/AmbientClipsConfigSO.cs

```diff
@@ -32,9 +32,11 @@ private bool TileChecks(GameObject player)
 			return tile != null && requiredTiles.Contains(tile);
 		}
 
-		public void PlayRandomClipLocally()
+		public string PlayRandomClipLocally()
 		{
-			_ = SoundManager.Play(ambientClips.PickRandom(), new Guid().ToString());
+			var token = Guid.NewGuid().ToString();
```

Comment:
thanks, finally someone does IDs with GUIDs instead of reinventing the wheel and creating a string generator

Reply:
I have a feeling I know what this is referencing to, lol